app/ytdl.py

```diff
@@ -138,7 +139,9 @@ async def update_status(self):
                 return
             self.tmpfilename = status.get('tmpfilename')
             if 'filename' in status:
-                self.info.filename = os.path.relpath(status.get('filename'), self.download_dir)
+                fileName = status['filename']

```

Comment:
The reason it was `status.get('filename')` and not `status['filename']` is so we don't crash if the status doesn't contain the filename. Can you change the code to handle this case?